katsdpingest/ingest_session.py

```diff
@@ -612,27 +620,58 @@ def _zero_counters(self) -> None:
         self.output_flagged_sensor.set_value(0, timestamp=now)
         self.output_vis_sensor.set_value(0, timestamp=now)
 
+    def _init_models(self) -> None:
+        self._rfi_mask_model = None
+        try:
+            base_url = self.telstate['sdp_model_base_url']
+            rfi_mask_model_key = self.telstate.join('model', 'rfi_mask', 'fixed')
+            rfi_mask_rel_url = self.telstate[rfi_mask_model_key]
+        except KeyError:
+            logger.info('Telescope state did not specify an RFI model')
```

Comment:
Or a missing base_url.

Reply:
I consider that to be part of the process of specifying an RFI model.